src/core/WebAppManagerConfig.cpp

```diff
@@ -80,11 +80,20 @@ void WebAppManagerConfig::initConfiguration()
 
 void WebAppManagerConfig::postInitConfiguration()
 {
-    if (access("/var/luna/preferences/debug_system_apps", F_OK) == 0) {
+    std::string dir;
+#if defined(HAS_AGL_SERVICE)
```

Comment:
This should be eventually reworked to use the more conventional IS_AGL. Or even better, have specific configure implementations for AGL and webOS. For now it should be ok.